gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/SelectableAvatar.kt

```diff
@@ -64,10 +80,25 @@ internal fun SelectableAvatar(
                 .clip(RoundedCornerShape(cornerRadius)),
         )
         when (loadingState) {
-            AvatarLoadingState.None -> LoadedOverlay({}, Modifier.align(Alignment.BottomEnd))
+            AvatarLoadingState.None -> LoadedOverlay({
+                moreOptionsPopupVisible = true
+            }, Modifier.align(Alignment.BottomEnd))
+
             AvatarLoadingState.Loading -> LoadingOverlay()
             is AvatarLoadingState.Failure -> FailureOverlay()
         }
+
+        if (moreOptionsPopupVisible) {
+            AvatarMoreOptionsPickerPopup(
+                anchorAlignment = Alignment.CenterHorizontally,
```

Comment:
It seems like sth is off with the alignment - it's always centered on the screen. Shouldn't it be centered to the anchor? 

It might not be related to this PR, but we might have to work on that calculation to fit this usage a bit better.

Reply:
I mentioned this in the PR description; I think we can iterate over the alignment in a different PR to keep this one small. But if you think that's a blocker for this one, I can work on it.

Reply:
No, no, not a blocker!

Reply:
👍 Ticket created -> #419